imod/tests/__init__.py

```diff
@@ -0,0 +1,32 @@
+# **************************************************************************
+# *
+# * Authors:     Federico P. de Isidro Gomez (fp.deisidro@cnb.csi.es) [1]
+# *
+# * [1] Centro Nacional de Biotecnologia, CSIC, Spain
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+from pyworkflow.tests import DataSet
+
+DataSet(name='tomo-em', folder='tomo-em',
```

Comment:
Shouldn't this dataset be defined at scipion-em-tomo?
Who uses this data set? Only iMod?

Reply:
Also in scipion-em-tomo and scipion-em-eman2

Reply:
Then...only in scipion-em-tomo would be necessary...I guess. Unless loading order affects this...actually, what is happening when the 3 defines it?